### PR TITLE
Add extra_vars support for Ansible

### PR DESCRIPTION
**Pull request by zmc**
Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
http://tracker.ceph.com/issues/11614

File: teuthology/task/ansible.py [hunk 1]
```diff
@@ -80,6 +82,9 @@ class Ansible(Task):
         inventory: /path/to/inventory
         playbook: /path/to/playbook.yml
         tags: my_tags
+        vars:
+            key1: value1
+            key2: value2
```
Comment (andrewschoen) [inline teuthology/task/ansible.py]: Could you add an example of a list and maybe a hash here as well?

```
vars:
    var1: string_value
    var2:
        - list_item
    var3:
        key: value
```

Comment (zmc) [inline teuthology/task/ansible.py]: Will do

File: teuthology/task/ansible.py [hunk 2]
```diff
@@ -60,6 +60,8 @@ class Ansible(Task):
                     not, we generate a temporary file to use.
         tags:       A string including any (comma-separated) tags to be passed
                     directly to ansible-playbook.
+        vars:       A dict of vars to be passed to ansible-playbook via the
+                    --extra-vars flag
```
Comment (andrewschoen) [inline teuthology/task/ansible.py]: Might be worth mentioning the possible values for a var are a yaml string, list or hash.

File: teuthology/task/ansible.py (hunk 1 above)
Comment (andrewschoen) [inline teuthology/task/ansible.py]: Awesome, thanks.

**Pull request merged**